crates/networking/p2p/net.rs

```diff
@@ -140,6 +141,7 @@ async fn discover_peers_server(
     signer: SigningKey,
     tx_broadcaster_send: broadcast::Sender<(tokio::task::Id, Arc<RLPxMessage>)>,
 ) {
+    debug!("Starting discovery server");
```

Comment:
We already have a tracing message that says `"Starting discovery service at {udp_addr}"` right before spawning this function, I would remove this one